xml/System.Collections.Concurrent/BlockingCollection`1.xml

```diff
@@ -1695,7 +1695,7 @@ This member is an explicit interface member implementation. It can be used only
       </Parameters>
       <Docs>
         <param name="item">The item to be added to the collection.</param>
-        <param name="millisecondsTimeout">The number of milliseconds to wait, or <see cref="F:System.Threading.Timeout.Infinite" /> (-1) to wait indefinitely.</param>
+        <param name="millisecondsTimeout">The number of milliseconds to wait for thread-safe access to the collection, or <see cref="F:System.Threading.Timeout.Infinite" /> (-1) to wait indefinitely.</param>
```

Comment:
I would change the wording slightly, it is [already given that any method in class](https://github.com/dotnet/dotnet-api-docs/blob/63308ebd05151c08a0fb364b06c12305875bcf1f/xml/System.Collections.Concurrent/BlockingCollection%601.xml#L76) guarantees thread-safe access.

```suggestion
        <param name="millisecondsTimeout">The number of milliseconds to wait for the collection to accept the item, or <see cref="F:System.Threading.Timeout.Infinite" /> (-1) to wait indefinitely.</param>
```